scheduler/admin.py

```diff
@@ -1,3 +1,10 @@
 from django.contrib import admin
-
+from .models import Professor, Student, Lesson, Group, Auditorium
 # Register your models here.
+
+
+admin.site.register(Student)
+admin.site.register(Professor)
+admin.site.register(Lesson)
+admin.site.register(Group)
+admin.site.register(Auditorium)
```

Comment:
Jak zacznie pylint działać to nakrzyczy za ten brak nowej linii na końcu :confused: